stores/src/main/java/com/issoft/training/stores/pars/DocPars.java

```diff
@@ -28,21 +30,61 @@ public void DocParse(String filepath) {
         }
     }
 
-    public String getOrderingValueByTag(String tagName) {
+    public List<String> getTagsValue() {
```

Comment:
you still create a lot of redundant methods and reread this doc several times, you should do the following, pseudocode. doc = new doc(file); map<> sortMap; for(node: nodes){sortMap.put(name, order);} return sortMap. Thats all!